valve-firmware/valve-firmware.spec

```diff
@@ -36,11 +36,7 @@ mv usr/lib/firmware/cs35l41-dsp1-spk-{cali.bin,cali.wmfw}.xz %{buildroot}/%{_pre
 tar -xvf %{_sourcedir}/steamdeck-dsp-*-any.pkg.tar.zst -C %{buildroot}/
 rm -f %{buildroot}/.BUILDINFO %{buildroot}/.MTREE %{buildroot}/.PKGINFO %{buildroot}/etc/wireplumber
 
-# Fix Steam Deck OLED Wi-Fi firmware with Linux >= 6.9 kernels.
-ln -s QCA206X %{buildroot}/usr/lib/firmware/ath11k/QCA2066
```

Comment:
Wouldn't this still be need for Fedora 40 and 39?

Reply:
Good point, thanks for bringing this up. So here's the odd situation for the latest `linux-firmware` package update:

- Fedora 41 = In testing repository
- Fedora 40 = In stable repository
- Fedora 39 = In testing repository

We should wait to merge this until it's been promoted to the stable repository (at least for Fedora 39).

Reply:
Fedora 39 has now promoted the new `linux-firmware` package to the stable updates repository. Fedora 41 has not but, considering it's a beta (and the updated package is staged in the testing repository), I think it's fine for us to merge this now.

- https://mirror.rackspace.com/fedora/updates/39/Everything/x86_64/Packages/l/
- https://mirror.rackspace.com/fedora/updates/testing/41/Everything/x86_64/Packages/l/